### PR TITLE
EX-363: test gnss-convertors rtcm 1230 fix

### PR DESCRIPTION
**Pull request by nsirola**
Pull in https://github.com/swift-nav/gnss-converters/pull/135

On bench with GPS+GLO only settings, this gives visibly better float solution than master.

To be tested with HITL with a suitable NTRIP stream (to be determined which ones send 1230 messages with indicator 1)

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.0.0-develop-2018100117-9-gf47f9abe
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=704262)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v2.0.0-develop-2018100117-9-gf47f9abe&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v2.0.0-develop-2018100117-9-gf47f9abe&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.0.0-develop-2018100117-9-gf47f9abe
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.0.0-develop-2018100117-9-gf47f9abe)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.0.0-develop-2018100117-9-gf47f9abe/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.0.0-develop-2018100117-9-gf47f9abe

File: package/gnss_convertors/gnss_convertors.mk [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-GNSS_CONVERTORS_VERSION = v0.3.64
+GNSS_CONVERTORS_VERSION = 6d5f0896caad82d16dcb401196997c39b8434f76
```
Comment (mbavaro) [inline package/gnss_convertors/gnss_convertors.mk]: this commit is https://github.com/swift-nav/gnss-converters/pull/135, can we merge and point to a release?

**Comment by nsirola** [comment]
According to their NTRIP source table, none of the BARD streams contain the 1230 message.

Thus testing on the MSM4 and MSM7 streams. I have no direct way of confirming the values of the 1230 message on those though.

If the RTCM data files in gnss-convertors are of any indication, Javad, Leica, Navcom, Topcon and Trimle have the indicator 0, while Septentrio is about the only one with 1.

**Comment by nsirola** [comment]
Though the logs from MSM7 have MSG_GLO_BIASES with all values already at 0, so the bug would not have affected them anyway.

**Comment by nsirola** [comment]
As mentioned in the gnss-converters PR https://github.com/swift-nav/gnss-converters/pull/135, I could not find any RTCM streams where bias indicator was 1 simultaneously with non-zero bias correction values, so the situation where this bug could manifest might be rare. No need to move pointers for this separately.

**Pull request closed without merging**